src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -1,107 +1,107 @@
+		assertTrue(driver.findElement(By.id("details-sed")).getText().contains("Safety Enhanced Design (SED)"));
+		assertTrue(driver.findElement(By.id("details-g1g2")).getText().contains("Successfully Tested G1/G2 Measures"));
+		assertTrue(driver.findElement(By.id("details-surveillance-activities")).getText().contains("Surveillance Activities"));
+		assertTrue(driver.findElement(By.id("details-additional-information")).getText().contains("Additional Information"));
+		
+	}
+
+	@When("^I open SED details accordion$")
+	public void i_open_SED_accordion() throws Throwable {
+		listingDetails_page.sed_details_accordion_open(driver);
+	}
+	
+	@Then("^usability report text should read as 'Full Usability Report'$")
+	public void usability_report_title_reads_as_Full_Usability_Report() throws Throwable {
+		
+	assertTrue(driver.getPageSource().contains("Full Usability Report"));
+	}
+
+	@Then("^intended user description header should show as Description of Intended Users$")
+	public void the_header_should_be_named_Description_of_Intended_Users() throws Throwable {
+	    
+	assertTrue(driver.getPageSource().contains("Description of Intended Users"));
+			
+	}
+
+	@Then("^the header name should read as 'Date SED Testing was Completed'$")
+	public void the_header_should_be_named_as_Date_SED_Testing_was_Completed() throws Throwable {
+		
+	assertTrue(driver.getPageSource().contains("Date SED Testing was Completed"));
+	}
+
+	@Then("^the download button title should read as 'Download SED Details'$")
+	public void the_download_button_title_should_read_as_Download_SED_Details() throws Throwable {
+		
+	assertTrue(driver.getPageSource().contains("Download SED Details"));
+	}
+	
+	@Then("^the criteria number should also include the criteria title$")
+	public void the_criteria_number_should_also_include_the_criteria_title() throws Throwable {
+		assertTrue(driver.getPageSource().contains("170.315 (a)(1): Computerized Provider Order Entry (CPOE) - Medications"));
+		assertTrue(driver.getPageSource().contains("170.315 (a)(2): CPOE - Laboratory"));
+		assertTrue(driver.getPageSource().contains("170.315 (a)(3): CPOE - Diagnostic Imaging"));
+		assertTrue(driver.getPageSource().contains("170.315 (a)(4): Drug-Drug, Drug-Allergy Interaction Checks for CPOE"));
+		assertTrue(driver.getPageSource().contains("170.315 (a)(5): Demographics"));
+		assertTrue(driver.getPageSource().contains("170.315 (a)(6): Problem List"));
+		assertTrue(driver.getPageSource().contains("170.315 (a)(8): Medication Allergy List"));
+		assertTrue(driver.getPageSource().contains("170.315 (a)(14): Implantable Device List"));
+		assertTrue(driver.getPageSource().contains("170.315 (b)(3): Electronic Prescribing"));
+		assertTrue(driver.getPageSource().contains("170.315 (a)(7): Medication List"));
+	}
+		
+	
+	@Given("^I am on the details page of listing \"([^\"]*)\"$")
+	public void i_open_SED_accordion(String arg3) throws Throwable {

```

Comment:
Why `arg3`? For that matter, why `arg1` and `arg2`? They should be meaningful names; for most of them I'd guess they should be something like `chplId`, though my guess (without looking) is the `arg1/arg2` will be `username/password`